src/components/Health/HealthDetails.tsx

```diff
@@ -35,33 +19,29 @@ export class HealthDetails extends React.PureComponent<Props, {}> {
     }
   }
 
-  renderPopover() {
+  render() {
     const health = this.props.health;
-    return (
-      <Popover id={this.props.id + '-health-tooltip'} title={this.props.headline}>
-        {health.items.map((item, idx) => {
-          return (
-            <div key={idx}>
-              <strong>
-                {this.renderStatus(item.status)}
-                {' ' + item.title + ': '}
-              </strong>
-              {item.text}
-              {item.children && (
-                <ul style={{ listStyleType: 'none', paddingLeft: 12 }}>
-                  {item.children.map((sub, subIdx) => {
-                    return (
-                      <li key={subIdx}>
-                        {this.renderStatus(sub.status)} {sub.text}
-                      </li>
-                    );
-                  })}
-                </ul>
-              )}
-            </div>
-          );
-        })}
-      </Popover>
-    );
+    return health.items.map((item, idx) => {
+      return (
+        <div key={idx}>
+          <strong>
```

Comment:
Do we need the key in this div ?? Can we replace this by <> </> ?

Reply:
I think the `key` is recommended as long as we're outputing JSX by mapping from an array. Else it raises a warning in JS console.